portal/assignment.py

```diff
@@ -200,3 +200,30 @@ def grade_submission():
                       res = cur.fetchall()
                       return redirect(url_for('teacher.sessions'))
     return redirect(url_for('teacher.courses'))
+
+
+@bp.route('/assignment/grades', methods=('GET', 'POST'))
```

Comment:
the route we've been using is '/assignments/' rather than '/assignment', and there is already an endpoint called '/assignments/grade', so making one called 'assignments/grades' would likely be confusing